nox/virtualenv.py

```diff
@@ -173,11 +206,15 @@ def _resolved_interpreter(self):
             xy_version = cleaned_interpreter
 
         path_from_launcher = locate_via_py(xy_version)
-
         if path_from_launcher:
             self._resolved = path_from_launcher
             return self._resolved
 
+        path_from_version_param = locate_using_path_and_version(xy_version)
```

Comment:
Any reason not to do this **before** the `if _SYSTEM != "Windows":` check on line 197? I.e. just checking for `python` on `PATH` and matching version against `python --version` could work fine on non-Windows operating systems.

Reply:
I don't want to put the python-in-path-check before the Windows check because, it being a stop-gap solution, then it would precede the python-launcher-check, which is preferrable if it exists. (py can find different python versions, if present, while python-in-path can only check whether the python in the path has the correct version. By putting our python-in-path-check above the py-check we would forfeit the possibility).

Personally, if something needs to be changed about this, I'd rather just remove the `Windows` check (as I understand it, it is not doing anything functionally, just skipping some (potentially) useless checks).

Reply:
That may actually be OK, but I'd rather do it in a separate PR. We could do that one in parallel with this one?

Reply:
Sounds good, left it as it is for now in this PR. Before starting another PR, I'll be sorta-kinda waiting first what else might pop up in this PR. 😃